Simplenote/Classes/InterlinkViewController.swift

```diff
@@ -54,7 +54,7 @@ extension InterlinkViewController {
         let anchorFrame = view.convert(anchor, from: nil)
         let editingRect = view.convert(viewport, from: nil)
 
-        let targetHeight = calculateHeight()
+        let targetHeight = calculateHeight(around: anchorFrame, in: editingRect)
         let targetTop = calculateTopLocation(for: targetHeight, around: anchorFrame, in: editingRect)
```

Comment:
From what I see these two methods do quite similar thing. 
In `calculateHeight(around: anchorFrame, in: editingRect)` we already know if there is more space above / below the `anchorFrame`. We calculate the available space again in a different way in `calculateTopLocation(for: targetHeight, around: anchorFrame, in: editingRect)`.
Do you think we can combine / optimise it?